src/cell/unsafe_cell.rs

```diff
@@ -0,0 +1,63 @@
+use crate::rt;
+
+/// A checked version of `std::cell::UnsafeCell`.
```

Comment:
It might be good if there was a brief comment here explaining why the API is not the same as `std::cell::UnsafeCell`?